libmscore/fret.cpp

```diff
@@ -39,6 +39,7 @@ static const ElementStyle fretStyle {
       { Sid::fretStrings,                        Pid::FRET_STRINGS            },
       { Sid::fretFrets,                          Pid::FRET_FRETS              },
       { Sid::fretNut,                            Pid::FRET_NUT                },
+      { Sid::fretMinDistance,                    Pid::MIN_DISTANCE            },
```

Comment:
This and the line change below are fixed in #5029, so could be removed from here.

Reply:
Yes, depending on which gets merged first, but that's for incorporating this!